internal/impl/shell.go

```diff
@@ -120,6 +120,14 @@ func shellPath(devbox *Devbox) (path string, err error) {
 	}
 	bashNixStorePath = string(out)
 
+	// install bashInteractive in nix/store without creating a symlink to local directory (--no-link)
+	cmd = exec.Command("nix", "build", bashNixStorePath, "--no-link")
+	cmd.Args = append(cmd.Args, nix.ExperimentalFlags()...)
+	err = cmd.Run()
+	if err != nil {
+		return "", errors.WithStack(err)
+	}
```

Comment:
Thoughts on using `devbox.addDevboxUtilityPackage` instead? It installs in a specific profile. (not sure if that works with `bashNixStorePath`)